store/errors.go

```diff
@@ -88,3 +90,21 @@ func NewErrNotFound(resource, id string) *ErrNotFound {
 func (e *ErrNotFound) Error() string {
 	return "resource: " + e.resource + " id: " + e.Id
 }
+
+// ErrOutOfBounds indicates that an invalid parameter, usually not related with the entity, was provided
```

Comment:
I would specifically say what it exactly is: an error when the total number of rows requested was beyond limits. It's not just any parameter.

Currently the description equates it to a generic invalid error not attached to an entity, which it's not.

Reply:
So I did not understand well, I thought it will be a generic error for invalid parameters